docs/plugins/aws/awsCompileScheduledEvents.md

```diff
@@ -0,0 +1,13 @@
+# awsCompileScheduledEvents
+
+This plugins compiles the function schedule event to to a CloudFormation resource.
+
+## How it works
+
+`awsCompileScheduleEvents` hooks into the `compileEvents` hook of the [deploy](/docs/plugins/core/deploy.md) plugin.
```

Comment:
Could you add the namespace to the lifecycle: `deploy:compileEvents` so that the reader knows where to find it and remove the "of the deploy plugin" part and add a link to the deploy plugin like it's done here:

https://github.com/serverless/serverless/blob/a76ed680460db45db3dd76b616425c8aaa96f6f3/docs/plugins/aws/awsCompileS3Events.md

This way when I merge the new docs the structure is the same.

Sorry for being a jerk here 🎅